src/Core/Configurations/RuntimeConfigProvider.cs

```diff
@@ -182,7 +182,7 @@ public async Task<bool> Initialize(string jsonConfig, string? graphQLSchema, str
 
         IsLateConfigured = true;
 
-        if (RuntimeConfigLoader.TryParseConfig(jsonConfig, out RuntimeConfig? runtimeConfig))
+        if (FileSystemRuntimeConfigLoader.TryParseConfig(jsonConfig, out RuntimeConfig? runtimeConfig))
```

Comment:
Should this be calling the instance method rather than the static method from the `_runtimeConfigLoader` field?

Reply:
fixed.